frontend/packages/lint-config/eslint/react.mjs

```diff
@@ -1,14 +1,14 @@
 import globals from 'globals';
-import pluginJs from '@eslint/js';
-import tseslint from 'typescript-eslint';
+import cdoBase from './base.mjs';
+import {configs as tseslintConfig} from 'typescript-eslint';
 import pluginReact from 'eslint-plugin-react';
 
 /** @type {import('eslint').Linter.Config[]} */
 export default [
   {files: ['**/*.{js,mjs,cjs,ts,jsx,tsx}']},
   {languageOptions: {globals: {...globals.node, ...globals.browser}}},
-  pluginJs.configs.recommended,
-  ...tseslint.configs.recommended,
+  ...cdoBase,
+  ...tseslintConfig.recommended,
```

Comment:
For my understanding, why don't we want this config in `node.mjs`?

Reply:
The design system doesn't use webpack to bundle, rather it uses esbuild + tsup so the circular dependency checker won't work there. The linter seemed like a reasonable alternative especially since we are able to have it error during development rather than the expensive build process.

The react config includes a jsx runtime, which isn't necessary for vanilla node (for example, scripts, backend libraries, build system, or backend services)

Reply:
Makes sense re: linter for circular dependency checker.

Sorry, not sure i'm following re: react config -- is there something jsx-specific about `tseslintConfig.recommended`, which is included in this config but not `node.mjs`?